src/components/Footer.css

```diff
@@ -73,16 +54,80 @@ footer a:hover {
 .footer-content p {
   text-align: left;
   width: calc(100% - 10.375rem);
+  margin-bottom: 0;
 }
 
 .languages {
   clear: both;
-  padding-left: 10px;
-  margin-left: 10px;
+  padding-left: 1.25rem;
+  margin-left: 1.25rem;
   border-left: 1px solid #666;
 }
 
 .languages .form-control {
   width: auto;
   padding: 0;
+}
+
+@media all and (min-width: 767px) {
+  footer .icon .footer-icon {
+    max-width: 18px;
+    margin-right: 5px;
+  }
+}
+
+@media all and (max-width: 767px) {
```

Comment:
everything looks good apart from certain resolutions where the description, logo and language selector are not correctly aligned

<img width="878" alt="screen shot 2018-07-11 at 19 34 07" src="https://user-images.githubusercontent.com/330324/42589551-a21fd7ba-8541-11e8-8124-d170a57e7a3d.png">

Reply:
Fixed!